galaxy/tools/deps/mulled/mulled_build.py

```diff
@@ -195,7 +199,7 @@ def mull_targets(
     involucro_args = [
         '-f', '%s/invfile.lua' % DIRNAME,
         '-set', "CHANNELS='%s'" % channels,
-        '-set', "TEST='%s'" % test,
+        '-set', "TEST=%s" % shlex_quote(test),
         '-set', "TARGETS='%s'" % target_str,
```

Comment:
I opted for the minimal change, but I guess `shlex.quote` could be applied for the other arguments too. Feel free to do so.